test/image/mocks/indicator_bignumber.json

```diff
@@ -0,0 +1,197 @@
+{
+  "data": [{
+      "domain": {
+        "x": [0, 0.33],
+        "y": [0, 0.33]
+      },
+      "title": {"text": "Accounts"},
+      "type": "indicator",
+      "mode": "number+delta",
+      "delta": {
+        "increasing": {
+          "color": "green",
+          "symbol": "+"
+        },
+        "valueformat": ".1%",
+        "reference": 400,
+        "relative": true
+      },
+      "gauge": {
+        "bgcolor": "rgba(255, 255, 255, 0.25)"
+      },
+      "max": 500,
```

Comment:
`vmax`?

Reply:
Cruft that we left over! Thanks for noticing :eyes:

Reply:
Removed in e6d8525ab07c56ca10bdbac6762b06d1d57bf830